_blog-src/_layouts/post.html

```diff
@@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
     <span style="color: #243446; margin-left: 15px; font-size: 13px"><i class="fa fa-tags"></i> {{ page.categories | array_to_sentence_string }}</span>
   </header>
 
+
+  {% if page.newUrl || page.oldUrl %}
+  <div class="name-change-alert" style="margin-top:15px;">
+    {% if page.newUrl %}
+    <!-- Old Post -->
+    <p>NOTE: A newer revision of this blogpost correct for <a href="{{ site.url }}{{ site.baseurl }}/{{ page.newUrl }}">apiman 1.2.x is available</a>. This version written for apiman 1.1.x, for which it remains valid.</p>
+    {% endif %}
+
+    {% if page.oldUrl %}
+    <!-- New Post -->
+    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
```

Comment:
How’s about:

This is a redux of an existing blogpost for apiman 1.2.x […]

Are you redoing all of them or just popular ones? If only popular ones I vote s/an existing/a popular/

> On 11 Jan 2016, at 21:15, Rachel Yordán <notifications@github.com> wrote:
> 
> In _blog-src/_layouts/post.html <https://github.com/apiman/apiman.github.io/pull/38#discussion_r49379648>:
> 
> > @@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
> >      <span style="color: #243446; margin-left: 15px; font-size: 13px"><i class="fa fa-tags"></i> {{ page.categories | array_to_sentence_string }}</span>
> >    </header>
> >  
> > +
> > +  {% if page.newUrl || page.oldUrl %}
> > +  <div class="name-change-alert" style="margin-top:15px;">
> > +    {% if page.newUrl %}
> > +    <!-- Old Post -->
> > +    <p>NOTE: A newer revision of this blogpost correct for <a href="{{ site.url }}{{ site.baseurl }}/{{ page.newUrl }}">apiman 1.2.x is available</a>. This version written for apiman 1.1.x, for which it remains valid.</p>
> > +    {% endif %}
> > +
> > +    {% if page.oldUrl %}
> > +    <!-- New Post -->
> > +    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
> @msavy <https://github.com/msavy> - Maybe something like this:
> 
> Note: This is a redux of an existing post for apiman 1.2.x. If you're still using apiman 1.1.x, you can refer to the older revision.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/apiman/apiman.github.io/pull/38/files#r49379648>.
>